scania-digital-design-system/sdds
# feat(divider): create webcomponent

**Describe pull-request**  
Web component version of the Divider. The component comes in a horizontal and vertical version, horizontal one being default. 
**Solving issue**  
Fixes:[AB#3336](https://dev.azure.com/scaniadigitaldesignsystem/Tegel/_sprints/taskboard/Design%20Tools%20and%20Systems/Tegel/Sprint%2052%20-%20IWD?workitem=3336)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Divider, Web Component
Test the controls and check the API.


**Suggested test steps**
- [ ] Direction, vertical and horizontal version
